src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImplTest.java

```diff
@@ -39,51 +45,30 @@ public void tearDown() throws Exception {
 
     @Test
```

Comment:
Given how relatively small and simple this test class is, it might be a good idea to go ahead and convert it over to using the JUnit 5 annotations.